internal/policy/verify.go

```diff
@@ -53,7 +53,7 @@ func VerifyRef(ctx context.Context, repo *git.Repository, target string) error {
 	}
 
 	// 2. Find latest entry for target
-	latestEntry, err := rsl.GetLatestEntryForRef(repo, target)
+	latestEntry, _, err := rsl.GetLatestEntryForRef(repo, target)
```

Comment:
These still use Entry...

Reply:
In this case, this is supposed to return an Entry and never an Annotation. So this is a correct use of "Entry".